lib/gridfs-stream/download.js

```diff
@@ -103,6 +127,36 @@ GridFSBucketReadStream.prototype.end = function(end) {
 };
 
 /**
+ * Marks this stream as destroyed (will never push another `data` event)
+ * and kills the underlying cursor. Will emit the 'end' event, and then
+ * the 'close' event once the cursor is successfully killed.
+ *
+ * @method
+ * @param {GridFSBucket~errorCallback} [callback] called when the cursor is successfully closed or an error occurred.
+ * @fires GridFSBucketWriteStream#close
+ * @fires GridFSBucketWriteStream#end
+ */
+
+GridFSBucketReadStream.prototype.destroy = function(callback) {
+  var _this = this;
+  this.push(null);
+  this.destroyed = true;
```

Comment:
This could be protected against multiple calls to `destroy()`, like it's done in the `WriteStream`.

`ReadableStream.push(null);` seems to be tolerant to multiple calls though, so it's not strictly required.